src/main/java/racingcar/service/RacingCarService.java

```diff
@@ -0,0 +1,67 @@
+package racingcar.service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import racingcar.model.Car;
+import racingcar.utlis.Convertor;
+import racingcar.validator.NameValidator;
+
+public class RacingCarService {
+
+	private List<Car> cars;
+
+	public RacingCarService(List<Car> cars) {
+		this.cars = cars;
+	}
+
+	public void playRound() {
+		cars.forEach(Car::decideMove);
+	}
+
+	public void saveCars(String inputCarNames) {
+		List<String> carNames = Arrays.asList(Convertor.separateNamesByDelimiter(inputCarNames));
+		NameValidator.checkNameList(carNames);
+		save(carNames);
+	}
+
+	private void save(List<String> carNames) {
+		for (String name : carNames) {
+			cars.add(new Car(name, 0));
+		}
+	}
```

Comment:
saveCars를 아래와 같이 여러번 동일한 인자로 호출하면 같은 이름을 가진 자동차가 목록에 담기겠네요.

```java
racingCarService.saveCars("A,B,C");
racingCarService.saveCars("A,B,C");
```

아래와 같이 덮어씌우는 건 어떨까요? 이렇게 하면 resetCars 메서드도 없어도 될 것 같아요.

```suggestion
	public void changeCars(String inputCarNames) {
		List<String> carNames = Arrays.asList(Convertor.separateNamesByDelimiter(inputCarNames));
		NameValidator.checkNameList(carNames);
		changeCars(carNames);
	}

	private void changeCars(List<String> carNames) {
	        this.cars = carNames.stream()
	                        .map(name -> new Car(name, 0))
	                        .collect(Collectors.toList());
	}
```